### PR TITLE
land: libshims: Fix typos

### PR DESCRIPTION
**Pull request by arizaini**
Change-Id: I67c15f1571a5a681ecd4acb2d478e170fb2a0742

### CODE REVIEW

File: BoardConfig.mk [hunk 1]
```diff
@@ -190,7 +190,7 @@ MALLOC_SVELTE := true
 BOARD_BOOTIMAGE_PARTITION_SIZE     := 67108864
 BOARD_CACHEIMAGE_PARTITION_SIZE    := 258998272
 BOARD_RECOVERYIMAGE_PARTITION_SIZE := 67108864
-BOARD_SYSTEMIMAGE_PARTITION_SIZE   := 3119513600 
+BOARD_SYSTEMIMAGE_PARTITION_SIZE   := 3119513600
```
Comment (nichcream) [inline BoardConfig.mk:193]: fair enough

File: BoardConfig.mk [hunk 2]
```diff
@@ -216,8 +216,8 @@ include device/qcom/sepolicy/sepolicy.mk
 BOARD_SEPOLICY_DIRS += $(DEVICE_PATH)/sepolicy
 
 # Shims
-TARGET_LD_SHIM_LIBS := /vendor/bin/mm-qcamera-daemon|libshim_pthreadts.so \
-    /system/bin/cameraserver|libshim_cameraservice.so
+TARGET_LD_SHIM_LIBS := /vendor/bin/mm-qcamera-daemon|libshim_pthreads.so \
+                       /system/bin/cameraserver|libshim_cameraservice.so
```
Comment (nichcream) [inline BoardConfig.mk:220]: ughh no, 4 spaces indent.

File: device.mk [hunk 3]
```diff
@@ -268,7 +268,7 @@ PRODUCT_PACKAGES += \
 # Libshims
 PRODUCT_PACKAGES += \
     libshim_cameraservice \
-    libshim_pthreadts
+    libshim_pthreads
```
Comment (nichcream) [inline device.mk:271]: pthread **T**ime**S**pec

File: libshims/Android.mk [hunk 4]
```diff
@@ -17,10 +17,11 @@ LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
 LOCAL_SRC_FILES := \
+    bionic/bionic_futex.cpp \
```
Comment (nichcream) [inline libshims/Android.mk:20]: Huh??

**Comment by arizaini** [comment]
I'll revert it...

**Pull request closed without merging**